### PR TITLE
WIP - Licensing - Make 32bit and 64 bit more explicit

### PR DESCRIPTION
**Pull request by gbiellem**
We had feedback via a desk case that the licensing doc should be more explicit about importing the license into the WOW3264Node of the registry

### CODE REVIEW

**Comment by gbiellem** [comment]
@indualagarsamy Is this what you had in mind when we discussed this on friday

File: nservicebus/licensing/license-management.md [hunk 1]
```diff
@@ -36,10 +37,12 @@ These following instructions cover installing the license file without using NSe
 - Create a new Multi-String Value (`REG_MULTI_SZ`) named `License`
 - Paste the contents of the license file you received from Particular Software.
 
+If this is a 64 bit operating system and you have chosen to add the license to `HKEY_LOCAL_MACHINE` then repeat the import process for the key `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\ParticularSoftware`
```
Comment (SimonCropp) [inline nservicebus/licensing/license-management.md]: remove you

File: nservicebus/licensing/license-management.md [hunk 2]
```diff
@@ -52,10 +55,7 @@ Set-ItemProperty -Path HKLM:\Software\ParticularSoftware -Name License -Force -V
 
 If modifying the registry directly using Registry Editor or a PowerShell script to update the license for ServiceControl, you will need to restart the ServiceControl service, as it only checks for its license information once at startup.
 
-NOTE: On a 64 bit operating system this script should not be run through the PowerShell(x86) console prompt, doing so will result in the license being imported into the 32 registry key. Please use a 64bit PowerShell session.
-
-NOTE: As of Version 4.5, both the `LicenseInstaller.exe` tool and the `install-NServiceBusLicense` PowerShell commandlet have been deprecated.
-
+NOTE: On a 64 bit operating system you should do this process in both the Powershell prompt and the PowerShell(x86) console prompt.  This will ensure the license is imported into both the 32 bit and 64 bit registry keys.
```
Comment (SimonCropp) [inline nservicebus/licensing/license-management.md]: remove you

Comment (SimonCropp) [inline nservicebus/licensing/license-management.md]: "do this process" perhaps "perform this action"?

File: nservicebus/licensing/license-management.md (hunk 1 above)
Comment (SimonCropp) [inline nservicebus/licensing/license-management.md]: "If this is a 64 bit operating system" seems strange. perhaps "on a..."

**Comment by gbiellem** [comment]
Thanks @SimonCropp  I've fixed it up based on your feedback.  
I want to get @indualagarsamy feeeback before it's merged though as this change was instigated by her.

**Comment by indualagarsamy** [comment]
@gbiellem - the clarifications make sense. I've also done a minor change to the order, i.e. moved the Powershell option higher, as it is the preferred approach. 
@gbiellem @SimonCropp - if you're happy with the changes, it's good to go.

**Pull request merged**